src/backend/function/Fusion.Resources.Functions/Functions/Notifications/ScheduledReportContentBuilderFunction.cs

```diff
@@ -95,84 +95,79 @@ private async Task BuildContentForTaskOwner(Guid azureUniqueId)
+        // Requests with start-date greater than three months form today
         var numberOfDepartmentRequestWithLessThanThreeMonthsBeforeStartAndNoNomination = departmentRequests
             .Count(x => x.OrgPositionInstance != null &&
                         x.State != null &&
                         !x.State.Equals("completed") && !x.Type.Equals("ResourceOwnerChange") &&
-                        (x.OrgPositionInstance.AppliesFrom < threeMonthsFuture &&
+                        (x.OrgPositionInstance.AppliesFrom < threeMonthsFromToday &&
                          x.OrgPositionInstance.AppliesFrom > today) && !x.HasProposedPerson);
 
 
-        //6.Requests with start-date > 3 months:
+        // Requests with start-date less than three months form today
         var numberOfDepartmentRequestWithMoreThanThreeMonthsBeforeStart = departmentRequests
-            .Count(x => x.OrgPositionInstance != null &&
+            .Count(x => x.Type != null &&
+                        x.OrgPositionInstance != null &&
                         x.State != null &&
                         !x.State.Contains("completed") && !x.Type.Equals("ResourceOwnerChange") &&
-                        x.OrgPositionInstance.AppliesFrom > threeMonthsFuture);
+                        x.OrgPositionInstance.AppliesFrom > threeMonthsFromToday);
 
-        //7.Average time to handle request (last 3 months): 
+        // Average time to handle request (last 3 months)
         var averageTimeToHandleRequest = CalculateAverageTimeToHandleRequests(departmentRequests);
 
-        //8.Allocation changes awaiting task owner action:
-        var numberOfAllocationchangesAwaitingTaskOwnerAction = GetchangesAwaitingTaskOwnerAction(departmentRequests);
+        // Allocation changes awaiting task owner action
+        var numberOfAllocationChangesAwaitingTaskOwnerAction = GetChangesAwaitingTaskOwnerAction(departmentRequests);
 
-        //9.Project changes affecting next 3 months
-        var numberOfChangesAffectingNextThreeMonths = GetAllChangesForResourceDepartment(personnelForDepartment);
+        // Project changes affecting next 3 months
+        var numberOfChangesAffectingNextThreeMonths = await GetAllChangesForResourceDepartment(personnelForDepartment);
 
-        //10.Allocations ending soon with no future allocation
+        // Allocations ending soon with no future allocation
         var listOfPersonnelWithoutFutureAllocations = FilterPersonnelWithoutFutureAllocations(personnelForDepartment);
 
-        //11.Personnel with more than 100 % workload
-        var listOfPersonnelWithTBEContent =
-            personnelForDepartment.Select(p => CreatePersonnelContentWithTotalWorkload(p));
-        var listOfPersonnelWithTBEContentOnlyMoreThan100PercentWorkload = listOfPersonnelWithTBEContent.Where(p => p.TotalWorkload > 100);
-
+        // Personnel with more than 100 % workload
+        var personnelEndingWithNoFutureAllocations = personnelForDepartment
```

Comment:
This looks like the wrong variablename. Should be "listOfPersonnelWithTBEContent" or something like that...
"personnelEndingWithNoFutureAllocations" relates to the previous point (10)